filebeat/harvester/util.go

```diff
@@ -6,12 +6,14 @@ import "github.com/elastic/beats/libbeat/common/match"
 const (
 	LogType   = "log"
 	StdinType = "stdin"
+	RedisType = "redis"
 )
 
-// ValidType is a list of all valid input types
+// List of valid input types
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported var ValidType should be of the form "ValidType ..."